homeassistant/const.py

```diff
@@ -88,6 +88,7 @@
 CONF_EVENT = 'event'
 CONF_FILE_PATH = 'file_path'
 CONF_FILENAME = 'filename'
+CONF_FORECAST_PERIODS = 'forecast_periods'
```

Comment:
Please leave this one in the `wunderground` file as I don't see other components needing this.